src/tribler-core/tribler_core/modules/metadata_store/tests/test_gigachannel_manager.py

```diff
@@ -290,16 +289,16 @@ def get_def(self):
     # Double conversion is required to make sure that buffers signatures are not the same
     mock_dl_list = [
         # Downloads for the personal channel
-        MockDownload(database_blob(bytes(my_chan_old_infohash)), personal_channel.dirname),
-        MockDownload(database_blob(bytes(personal_channel.infohash)), personal_channel.dirname),
+        MockDownload(bytes(my_chan_old_infohash), personal_channel.dirname),
+        MockDownload(bytes(personal_channel.infohash), personal_channel.dirname),
         # Downloads for the updated external channel: "old ones" and "recent"
-        MockDownload(database_blob(b'12331244'), chan2.dirname),
-        MockDownload(database_blob(bytes(chan2.infohash)), chan2.dirname),
+        MockDownload(b'12331244', chan2.dirname),
+        MockDownload(bytes(chan2.infohash), chan2.dirname),
         # Downloads for the unsubscribed external channel
-        MockDownload(database_blob(b'1231551'), chan3.dirname),
-        MockDownload(database_blob(bytes(chan3.infohash)), chan3.dirname),
+        MockDownload(b'1231551', chan3.dirname),
+        MockDownload(bytes(chan3.infohash), chan3.dirname),
```

Comment:
It seems likely that these `bytes` calls could also be removed. Could you double-check this is still needed?

Reply:
Nice find!